docs/root/version_history/current.rst

```diff
@@ -125,6 +125,7 @@ New Features
 * access log: added the new response flag `NC` for upstream cluster not found. The error flag is set when the http or tcp route is found for the request but the cluster is not available.
 * access log: added the :ref:`formatters <envoy_v3_api_field_config.core.v3.SubstitutionFormatString.formatters>` extension point for custom formatters (command operators).
 * access log: added support for cross platform writing to :ref:`standard output <envoy_v3_api_msg_extensions.access_loggers.stdoutput.v3.StdoutputAccessLog>` and :ref:`standard error <envoy_v3_api_msg_extensions.access_loggers.stderror.v3.StderrorAccessLog>`.
+* access log: added support for json format in the :ref:`formatters <envoy_v3_api_field_config.core.v3.SubstitutionFormatString.formatters>` extension point.
```

Comment:
This phrasing sounds a bit weird since it sounds like we're adding support for emitting JSON from the formatters or something, maybe something like "formatters provided via the <whatever> extension point can now be used with json format"?

Reply:
Oh hold on, there was no release since formatters were added so we can just skip adding an entry and the existing entry (two lines above) would cover us (since now it supports all log formats).

Reply:
Resolving this since I think it's OK to just drop the new entry.